src/screens/V4VProvidersAlbyLoginScreen.tsx

```diff
@@ -37,11 +39,43 @@ export class V4VProvidersAlbyLoginScreen extends React.Component<Props, State> {
   }
 
   render() {
+    const { navigation } = this.props
     const { url } = this.state
 
     return (
       <View style={styles.wrapper}>
-        <WebView overScrollMode='never' removeClippedSubviews source={{ uri: url }} style={{ opacity: 0.99 }} />
+        <WebView
+          dataDetectorTypes={['link']}
+          originWhitelist={['https://', PV.DeepLinks.prefix]}
+          onShouldStartLoadWithRequest={(request) => {
+            // This is mostly duplicated in PodcastsScreen deep links.
```

Comment:
Maybe extract it to a helper function then? It's not the simplest block of code. Also maybe we can utilize the `qs` library since we have it?

Reply:
Good catch. I read somewhere it's important to use helper functions in component props for performance reasons.